packages/design-system/src/components/contextMenu/contextMenuProvider/provider.tsx

```diff
@@ -17,23 +17,29 @@
  * External dependencies
  */
 
-import PropTypes from 'prop-types';
 import { useCallback, useState } from '@googleforcreators/react';
+import type { PropsWithChildren } from 'react';
 /**
  * Internal dependencies
  */
 import ContextMenuContext from './context';
 
+interface ContextMenuProviderProps {
+  isIconMenu?: boolean;
+  isHorizontal?: boolean;
+  onDismiss: (evt?: Event) => void;
+}
+
 export default function ContextMenuProvider({
   children,
   isIconMenu = false,
   isHorizontal = false,
   onDismiss,
-}) {
-  const [focusedId, setFocusedId] = useState(-1);
```

Comment:
Was there a reason for the `-1` before?

Reply:
I couldn't find any reference to it at all actually. I think it was just a placeholder that would never match a real id.